core/server/common/src/main/java/alluxio/master/journal/ufs/UfsJournal.java

```diff
@@ -146,7 +146,9 @@ protected static UnderFileSystemConfiguration getJournalUfsConf() {
    */
   public UfsJournal(URI location, Master master, long quietPeriodMs,
       Supplier<Set<JournalSink>> journalSinks) {
-    this(location, master, UnderFileSystem.Factory.create(location.toString(), getJournalUfsConf()),
```

Comment:
Calling `getJournalUfsConf()`, or doing the same operations as that method (https://github.com/Alluxio/alluxio/blob/v2.4.1/core/server/common/src/main/java/alluxio/master/journal/ufs/UfsJournal.java#L131), is important. Is the new behavior equivalent to the previous call to `getJournalUfsConf()`?

Reply:
Thanks for this suggestion, I'm doing the same operations `getJournalUfsConf()`  in the AbstractUfsManager#getJournal